migrations/sqls/20190310135013-inital-ddl-up.sql

```diff
@@ -0,0 +1,96 @@
+/* Replace with your SQL commands */
+/*dev_trap DDL */
+
+/*TraPO - Data defintion language */
+
+--USE dev_trapo;
+
+/*Create person.person table */
+CREATE TABLE IF NOT EXISTS person (
+  personid int NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  persontype nvarchar(32) NOT NULL, /*PersonType contraint at front end */
+  firstname nvarchar(255) NOT NULL,
+  lastname nvarchar(255) NOT NULL,
+  dob date NOT NULL,
+  gender nvarchar(32) NOT NULL,  /*Gender constraint at front end */
+  dateadded DATETIME DEFAULT CURRENT_TIMESTAMP,
+  datemodified DATETIME DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  addedby int NOT NULL /*Self Reference to Person ID*/,
+  modifiedby int NOT NULL /*Self Reference to Person ID*/,
+  FOREIGN KEY (addedby) REFERENCES person(personid),
```

Comment:
Shouldn't `addedby` refer to the `users` table, not the `person` table? Aren't people added by CU users, not by other CU clientele?